src/Illuminate/Session/Store.php

```diff
@@ -6,6 +6,7 @@
 use Illuminate\Contracts\Session\Session;
 use Illuminate\Support\Arr;
 use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Date;
```

Comment:
This is already been used in `illuminate/session` so adding this should be safe.